tests/integration/test_remoted/test_agent_communication/conftest.py

```diff
@@ -0,0 +1,21 @@
+"""
+ Copyright (C) 2015-2043, Wazuh Inc.
+ Created by Wazuh, Inc. <info@wazuh.com>.
+ This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+"""
+
+
+import pytest
+
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.modules.analysisd.patterns import ANALYSISD_STARTED
+from wazuh_testing.utils import callbacks
+from wazuh_testing.tools.monitors import file_monitor
+
+
+@pytest.fixture(scope='module')
+def waiting_for_analysisd_startup(request):

```

Comment:
Why do you check this in a `remoted` test? Couldn't you check another log from `remoted`?

Reply:
I spoke with Tomas and an agreement was reached not to do it due to how the tests were designed.